test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/FunctionApps/java/EventHub/extensions.csproj

```diff
@@ -5,7 +5,7 @@
 	<DefaultItemExcludes>**</DefaultItemExcludes>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Kafka" Version="100.100.100-pre" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Kafka" Version="" />
```

Comment:
If we keep the version empty, which version would it pick up.

Reply:
It would pick the latest available version. This is done to make the independent build successful as well - without building the 100.100.100-pre nuget for the function build.